From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ram Muthiah <rammuthiah@google.com>
Date: Fri, 17 Jan 2020 16:20:14 -0800
Subject: ANDROID: build config for cuttlefish ramdisk

Both x86 and arm64 are covered in this change.
The ramdisk and ko's are put in the DIST_DIR by the
BUILD_INITRAMFS option.

Bug: 139431025
Test: Local build and boot of cf
Signed-off-by: Ram Muthiah <rammuthiah@google.com>
Change-Id: I82a410ea3cf9b9177a1c8f2186d88344b9a63488
---
 cuttlefish.fragment | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)
 create mode 100644 cuttlefish.fragment

diff --git a/cuttlefish.fragment b/cuttlefish.fragment
new file mode 100644
--- /dev/null
+++ b/cuttlefish.fragment
@@ -0,0 +1,20 @@
+CONFIG_CPUFREQ_DUMMY=m
+CONFIG_BINFMT_MISC=m
+CONFIG_VSOCKETS=m
+CONFIG_VIRTIO_VSOCKETS=m
+CONFIG_CAN=m
+CONFIG_CAN_VCAN=m
+CONFIG_GNSS_CMDLINE_SERIAL=m
+CONFIG_VIRTIO_BLK=m
+CONFIG_VIRTIO_NET=m
+CONFIG_VIRT_WIFI=m
+CONFIG_SERIAL_OF_PLATFORM=m
+CONFIG_VIRTIO_CONSOLE=m
+CONFIG_HW_RANDOM_VIRTIO=m
+CONFIG_DRM_VIRTIO_GPU=m
+CONFIG_SND_INTEL8X0=m
+CONFIG_USB_DUMMY_HCD=m
+CONFIG_RTC_DRV_TEST=m
+CONFIG_VIRTIO_PCI=m
+CONFIG_VIRTIO_INPUT=m
+CONFIG_VIRTIO_MMIO=m
